metrics/web/lib/common/presentation/navigation/state/navigation_notifier.dart

```diff
@@ -15,7 +15,13 @@ typedef MetricsPagePredicate = bool Function(MetricsPage);
 
 /// A [ChangeNotifier] that manages navigation.
 class NavigationNotifier extends ChangeNotifier {
-  /// A [NavigationState] used to interact with the current navigation state.
+  /// A [RouteConfiguration] that represents the loading page.
+  static const RouteConfiguration _loadingRoute = RouteConfiguration.loading();
+
+  /// A [RouteConfiguration] that represents the login page route.
+  static final RouteConfiguration _loginRoute = RouteConfiguration.login();
```

Comment:
📱 
Not sure we should use route configurations like this in the notifier. Maybe it makes sense to think about introducing the route constants to the application like the `MetricsRoutes`. The idea is to keep the "default" configurations separately. There are a few benefits: 
- if some route requires a query parameter by default - we can provide the default route with this parameter and easily maintain other defaults;
- it possibly improves code readability and makes it clearer.